ros_buildfarm/templates/__init__.py

```diff
@@ -75,6 +76,7 @@ def expand_template(template_name, data, options=None):
     try:
         interpreter = CachingInterpreter(output=output, options=options)
         for template_hook in template_hooks or []:
+            assert isinstance(template_hook, Hook)
```

Comment:
This is really just here to please the linters.

We can't use `noqa` because pyflakes doesn't respect that, and we can't reference the type as an actual `typing` directive because pep8 doesn't support that syntax.

Reply:
:rage4: